rpc/src/methods.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+//! Module contains RPC method handlers for Full Node JSON-RPC interface
+use crate::views::AccountView;
+use anyhow::ensure;
+use anyhow::Result;
```

Comment:
why not use anyhow::{ensure, Result}; it seems pretty common to merge these as much as possible.

Reply:
lol, good catch. I thought for some reason that linter merges them together automatically